x-pack/plugins/infra/server/lib/alerting/log_threshold/register_log_threshold_rule_type.ts

```diff
@@ -41,13 +42,6 @@ const conditionsActionVariableDescription = i18n.translate(
   }
 );
 
-const groupByActionVariableDescription = i18n.translate(
```

Comment:
Out of curiosity: Why was this message moved to common?

Reply:
The description of messages across rules is slightly different. And for log threshold rule, the messages are in same file, whereas for infra rules, they are in common folder. Since the `legacyGroupActionVariableDescription` is used in both Metric and Log threshold rules, I put it in common folder. The `groupByActionVariableDescription` from Log threshold rule is no longer used, so I removed it.